LINK/etc/profile.d/evm.sh

```diff
@@ -4,3 +4,13 @@
 alias ap='/usr/bin/appliance_console'
 alias vmdb='cd /var/www/miq/vmdb'
 alias appliance='[[ -n ${APPLIANCE_SOURCE_DIRECTORY} ]] && cd ${APPLIANCE_SOURCE_DIRECTORY}'
+
+# Tail Logs:
+function tailmiq() #If no value is given with tailmiq it defaults to the evm.log
+{
+  tail -f /var/www/miq/vmdb/log/${1:-evm}.log
+}
+alias tailpglog='tail -f $APPLIANCE_PG_DATA/pg_log/postgresql.log'
+
+# Appliance Status:
+alias apstatus='echo "EVM Status:";service evmserverd status;echo " ";echo "HTTP Status:";service httpd status'
```

Comment:
We probably want to convert this to systemctl in a follow up PR.  We might also want to add `memcached`, postgresql via the ever changing [APPLIANCE_PG_SERVICE env variable](https://github.com/ManageIQ/manageiq-appliance/blob/d356f7702f097f5bf131c578b1c31ae9d7ad5920/LINK/etc/default/evm_postgres#L7)